### ISSUE TITLE
dns_a_record_set and dns_aaaa_record_set crashes

### ISSUE THREAD

**Issue by andreaso** [terraform-providers/terraform-provider-dns#15]
### Terraform Version
- Terraform v0.10.7
- terraform-provider-dns 1.0.0

### Affected Resource(s)
Please list the resources as a list, for example:
- dns_a_record_set
- dns_aaaa_record_set

### Terraform Configuration Files
```hcl
provider "dns" {
  update {
    server        = "85.119.83.73"
    key_name      = "wensicia-infra"
    key_algorithm = "hmac-sha256"
    key_secret    = "<redacted>"
  }
}

resource "dns_a_record_set" "test1" {
  zone = "arrakis.se."
  name = "test1.arrakis.se."
  addresses = [
    "127.0.0.1",
  ]
}
```

### Debug Output
https://gist.github.com/andreaso/c5562dcb288de54de893ecb8430410c7

### Panic Output
https://gist.github.com/andreaso/f248319babb6774ba98536dc7b7d5dc1

### Expected Behavior
I expected an A record to be created.

### Actual Behavior
Terraform crashed.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

Getting the same crash with `dns_aaaa_record_set`.

**Comment by kradalby** [comment]
+1

**Comment by filterfish** [comment]
For anyone else having this problem the patch on #36 applied to `HEAD` (f2f3da9) fixes it.

**Comment by katbyte** [comment]
Should be fixed by #36
